src/notify.js

```diff
@@ -71,6 +83,11 @@ Notify.isSupported = function(perm) {
         throw new Error('You must only call this before calling Notification.requestPermission(), otherwise this feature detect would trigger an actual notification!');
     }
 
+    if (Notify.serviceWorkerSupported()) {
```

Comment:
well, a simple solution to the problem would be:

```javascript
Notify.isSupported = function(usingServiceWorker = false) {
    ...
};
```

This would not break chrome